corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_view.js

```diff
@@ -0,0 +1,117 @@
+/*global FormplayerFrontend */
+
+FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, FormplayerFrontend, Backbone, Marionette) {
+    MenuList.MenuView = Marionette.ItemView.extend({
+        tagName: "tr",
+        template: "#menu-view-item",
+
+        events: {
+            "click": "rowClick",
+        },
+
+        getTemplate: function () {
+            if (this.model.attributes.audioUri) {
```

Comment:
still think we should go back and change all these references to `attributes`, but not blocking